addons/event/static/src/client_action/event_registration_summary_dialog.xml

```diff
@@ -6,25 +6,35 @@
        <Dialog size="'md'" title="'Home'">
             <div class="row">
                 <div class="col-lg-10 w-100 fs-2">
-                    <div t-if="registrationStatus.value === 'confirmed_registration'" class="alert alert-success text-center" role="alert">
-                        <i class="fa fa-solid fa-check-circle me-2"/>
-                        <span>Successfully registered!</span>
+                    <div t-if="registrationStatus.value === 'confirmed_registration'" class="alert alert-success d-flex justify-content-between align-items-center" role="alert">
+                        <div class="o_event_barcode_dialog_status d-flex flex-fill justify-content-center ms-0 ms-md-5">
+                            <i class="fa fa-solid fa-check-circle me-2 align-self-center"/>
+                            <span>Successfully registered!</span>
+                        </div>
+                        <button type="button" class="btn btn-link ms-3" t-on-click="undoRegistration">
+                            Undo
```

Comment:
add in pot

Reply:
They already are present in the pot file :face_with_peeking_eye:

Reply:
The _undo_? :p I do not see it